src/ORM/Association.php

```diff
@@ -101,6 +101,13 @@
     protected $_className;
 
     /**
+     * The registry alias of the target table object
+     *
+     * @var string
```

Comment:
@dereuromark I'm dropping the idea as it leaves nowhere good. See comments above.